x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_indices.tsx

```diff
@@ -80,21 +84,24 @@ export const EngineIndices: React.FC = () => {
     },
     type: 'icon',
   };
+
   const columns: Array<EuiBasicTableColumn<EnterpriseSearchEngineIndex>> = [
     {
-      field: 'name',
       name: i18n.translate('xpack.enterpriseSearch.content.engine.indices.name.columnTitle', {
         defaultMessage: 'Index name',
       }),
-      render: (name: string) => (
-        <EuiLinkTo
-          data-test-subj="engine-index-link"
-          to={generateEncodedPath(SEARCH_INDEX_PATH, { indexName: name })}
-        >
-          {name}
-        </EuiLinkTo>
-      ),
-      sortable: true,
+      render: ({ health, name }: EnterpriseSearchEngineIndex) =>
+        health === 'unknown' ? (
+          name
```

Comment:
Can we grey out the text in the rest of the row as well, like in the [Figma](https://www.figma.com/file/IupYoBQ7qMh0dMBGGz2kGq/Introducing-Engines?node-id=1196%3A230707&t=FuQAxUFh4JrIw2wF-0)?

Reply:
i'll see what i can do. the Eui component doesn't support this directly